src/itempicker.cpp

```diff
@@ -75,6 +98,46 @@ QList<QQuickItem*> ItemPicker::pickItem(QMouseEvent* event, QQuickItem* item) {
         continue;
       }
 
+      list.append(pickItem(event, child));
+    }
+  }
+
+  return list;
+}
+
+QList<QQuickItem*> ItemPicker::pickItem(QTouchEvent* event, QQuickItem* item) {
+  QList<QQuickItem*> list;
+
+  if (event->touchPoints().length() != 1) {
```

Comment:
Also this needs to be checked. Do we want to support multi-touch interactions?

Reply:
Let's start with this for now. If folks are prevented from clicking the qmlEmitter with two fingers instead of one than maybe we reconsider.